### PR TITLE
added auto hide timeout

### PR DESCRIPTION
**Pull request by cristianoalves92**
I added an option to auto hide alert

### CODE REVIEW

File: CDAlertView/Classes/CDAlertView.swift [hunk 1]
```diff
@@ -162,6 +162,8 @@ open class CDAlertView: UIView {
 
     public var hideAnimationDuration: TimeInterval = 0.5
 
+    public var autoHideTime: TimeInterval!
```
Comment (candostdagdeviren) [inline CDAlertView/Classes/CDAlertView.swift]: What about making it an optional?

Comment (cristianoalves92) [reply CDAlertView/Classes/CDAlertView.swift]: Yes can be optional

File: CDAlertView/Classes/CDAlertView.swift [hunk 2]
```diff
@@ -290,8 +294,18 @@ open class CDAlertView: UIView {
         popupViewInitialFrame = popupView.frame
 
         completionBlock = completion
+        
+        if(self.autoHideTime != nil) {
```
Comment (candostdagdeviren) [inline CDAlertView/Classes/CDAlertView.swift]: Sorry for not mentioning at first but I just realized that we're not using parentheses in `if` cases, could you remove that one to have same style in everywhere. Also could you change it to optional unwrapping like
`if let autoHideTime = self.authoHideTime { ...  }` Than, I'll test and merge

Comment (cristianoalves92) [reply CDAlertView/Classes/CDAlertView.swift]: 👍

**Pull request merged**